### PR TITLE
[All Hosts] (ribbon) document InvalidRibbonDefinition error

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

File: docs/testing/application-specific-api-error-handling.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ The following tables list the errors that application-specific APIs may return.
 |`InvalidOperation`|The operation attempted is invalid on the object.|*None.* |
 |`InvalidReference`|This reference is not valid for the current operation.|*None.* |
 |`InvalidRequest`  |Cannot process the request.|*None.* |
+|`InvalidRibbonDefinition` | Office has been given an invalid ribbon definition | This error is thrown if an invalid [RibbonUpdateObject](/javascript/api/office/office.ribbonupdaterdata) is passed to the [Office.ribbon.requestUpdate()](/javascript/api/office/office.ribbon#office-office-ribbon-requestupdate-member(1)) method.
```
Comment (AlexJerabek) [inline docs/testing/application-specific-api-error-handling.md:79]:
```suggestion
|`InvalidRibbonDefinition` | Office has been given an invalid ribbon definition. | This error is thrown if an invalid [RibbonUpdateObject](/javascript/api/office/office.ribbonupdaterdata) is passed to the [Office.ribbon.requestUpdate()](/javascript/api/office/office.ribbon#office-office-ribbon-requestupdate-member(1)) method.  
```

**Review by AlexJerabek — APPROVED** [review]
Approving with a small suggestion.

File: docs/testing/application-specific-api-error-handling.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/testing/application-specific-api-error-handling.md]: Done

**Comment by Rick-Kirkham** [comment]
@AlexJerabek I found another missing code. Please re-review.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**